packages/aws-cdk-lib/aws-stepfunctions-tasks/test/bedrock/invoke-model.test.ts

```diff
@@ -203,6 +203,52 @@ describe('Invoke Model', () => {
     });
   });
 
+  test('invoke model allows input and output json path', () => {
+    const stack = new cdk.Stack();
+    const model = bedrock.ProvisionedModel.fromProvisionedModelArn(stack, 'Imported', 'arn:aws:bedrock:us-turbo-2:123456789012:provisioned-model/abc-123');
+
+    const task = new BedrockInvokeModel(stack, 'Invoke', {
+      model,
+      inputPath: sfn.JsonPath.stringAt('$.prompt'),
+      outputPath: sfn.JsonPath.stringAt('$.prompt'),
+    });
+
+    new sfn.StateMachine(stack, 'StateMachine', {
+      definitionBody: sfn.DefinitionBody.fromChainable(task),
+    });
+
+    // THEN
+    expect(stack.resolve(task.toStateJson())).toEqual({
```

Comment:
Can we test that the right policies are added to the `policyStatement` whenever `inputPath` or `outputPath` are set

Reply:
sure, will add a test for the policy statement check.

Reply:
added test along with squashish commits